### PR TITLE
test

### PR DESCRIPTION
**Pull request by StormyW618**
Milestone #1 Completed

### CODE REVIEW

**Review by bbmcmann — COMMENTED** [review]
Hello! One quick note on your milestone 1 submission. I am assuming you are developing in a mac or linux environment and didn't run into this error, but your first file < !DOCTYPE html >.html is actually an illegal filename on windows so I am unable to even see it. This is a funny error I've never run into so thought I would let you know. I'm assuming the filename was a typo since the rest of your files have good names, but it would be best to name this index.html or something more meaningful and portable.
One other note, when making a pull request it helps to name it something like 'milestone 1 completed' or 'milestone #' so we know what we are reviewing. Just helps us stay a bit more organized and prepare you for pull requests on project teams.
Cole is able to review your code in a mac environment so that will be completed sometime this weekend.

File: <!DOCTYPE html>.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cooking up a Storm</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+      <header class="header-background">
+        <h1>Cooking up a Storm</h1>
+        <nav class = "navibar">
+            <a href= "<!DOCTYPE html>.html">HOME</a>
```
Comment (ctperry0301) [inline <!DOCTYPE html>.html:14]: This is a weird file name. It probably doesn't cause any issues, but naming it something like home.html or index.html would be better for readability.

File: <!DOCTYPE html>.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      <main>
+          <h1>Five Star Breakfast</h1>
+          <h2>
+          <a href = "french.html"> French Toast<a>
+            <div class="search-container">
+                <img class = "search-item" src = "https://mccormick.widen.net/content/z8hiapwdst/original/quick_and_easy_french_toast_new_800x800.webp">
+            </div>
+            </h2>
+          <h2><a href = "pancake.html"> Pancakes<a>
+            <div class="search-container">
```
Comment (ctperry0301) [inline <!DOCTYPE html>.html:27]: I noticed that you use class here and a few other places, but you don't make use of your classes in your styles anywhere

File: pancake.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          <ol>
+              <li>Pour 1 cup of pancake mix into medium sized bowl</li>
+              <li>Pour in about 2/3 cup of water into the medium sized bowl along with the pancake mix</li>
+              <li>Stir until fully blended</li>
+              <li>Pour pancake mix into frying pan on medium heat</li>
+              <li>Cook on each side for 1-1 1/2 minutes </li>
+              <li>Serve with syrup and/or butter</li>
+            </ol>
+      </p>
+	<script src="index.js"></script>
```
Comment (ctperry0301) [inline pancake.html:37]: Get rid of unused script tags unless you plan to do something with them

**Review by ctperry0301 — COMMENTED** [review]
Nice job! Your code meets all the requirements for milestone 1. A few notes I have are that you should name your pull requests something a little bit more relevant. Like Milestone 1 completed. Also, I noticed that you have a lot of classes and tags that aren't used. Removing them will make your code more organized. Finally, <!DOCTYPE html>.html is not a good filename. I would suggest re-naming this to index.html or home.html

**Pull request closed without merging**